Source/KakapoDB.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  KakapoDB.swift
+//  KakapoExample
+//
+//  Created by Joan Romano on 31/03/16.
+//  Copyright © 2016 devlucky. All rights reserved.
+//
+
+import Foundation
+
+protocol KStorable {
+    var id: Int { get }
```

Comment:
I think yes since you are asserting if the id is wrong, also how do you deal with it with the scenarios?
Thread safety for the id (if we keep it incremental) is a must have for me, I guess it could be common to have threading within the app that shouldn't be fUcked by our lib